reference/api/indexes.md

```diff
@@ -76,7 +109,7 @@ Get information about an [index](/learn/core_concepts/indexes.md). The index [`u
 
 <RouteHighlighter method="POST" route="/indexes"/>
 
-Create an [index](/learn/core_concepts/indexes.md). This endpoint accepts two arguments: `uid` and `primaryKey`.
+Create an index. This endpoint accepts two arguments: `uid` and `primaryKey`.
```

Comment:
This doesn't seem necessary since both arguments are explained in the body table below.

```suggestion
Create an index.
```

I would also recommend deleting the below line relating to primary key inference. If the user wants to read more about primary key, they can do so by following the link in the body table.

I'm torn about whether to leave or remove the note about implicit index creation. It is already documented in the [indexes core concepts guide](https://docs.meilisearch.com/learn/core_concepts/indexes.html#index-creation), and doesn't directly relate to the create index endpoint. Leaning towards removing it since it seems needlessly distracting.